src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/DocumentColor/DocumentColorHandler.ts

```diff
@@ -23,16 +25,16 @@ export class DocumentColorHandler {
 
     public register() {
         // tslint:disable-next-line: no-floating-promises
-        this.serverClient.onRequestWithParams<SerializableTextDocumentIdentifier, vscode.ColorInformation[], any>(
+        this.serverClient.onRequestWithParams<SerializableDocumentColorParams, SerializableColorInformation[], any>(
             this.documentColorRequestType,
-            async (request: SerializableTextDocumentIdentifier, token: vscode.CancellationToken) => this.provideHtmlDocumentColors(request, token));
+            async (request: SerializableDocumentColorParams, token: vscode.CancellationToken) => this.provideHtmlDocumentColors(request, token));
     }
 
     private async provideHtmlDocumentColors(
-        documentColorParams: SerializableTextDocumentIdentifier,
+        documentColorParams: SerializableDocumentColorParams,
         cancellationToken: vscode.CancellationToken) {
         try {
-            const razorDocumentUri = vscode.Uri.parse(`vscode:${documentColorParams.uri}`, true);
+            const razorDocumentUri = vscode.Uri.parse(`${documentColorParams.textDocument.uri}`, true);
```

Comment:
The interpolation seems redundant now. Is it a TypeScript style thing to just always do this? Does the TS compiler optimize it away?

Reply:
Good catch, it is indeed redundant. Fixed, thanks!